components/SearchBar/index.js

```diff
@@ -108,9 +114,10 @@ function SearchBar({ search, onSearch }) {
           </Styled.FilterContainer>
           <Styled.FilterContainer>
             <Styled.FilterItem>
-              <label>Amount From</label>
+              <label for="amount_from">Amount From</label>
```

Comment:
[We need to use `htmlFor` in React](https://legacy.reactjs.org/docs/dom-elements.html#htmlfor).